### PR TITLE
Add initial implicit capture proposal

### PR DESCRIPTION
**Pull request by RByers**
Adds an initial description of chromium's "implicit capture" behavior as discussed in w3c/pointerevents#8.

### CODE REVIEW

**Comment by RByers** [comment]
Note that this is a pull request against a new "reduce-hit-tests" branch as discussed in w3c/pointerevents#8.  I imagine we'll want to continue to iterate on related changes in the branch (for example, this doesn't yet contain the fix for w3c/pointerevents#566).  So I don't think we need to worry about this being perfect from the start, I just want to get an initial framework in place for describing the chromium implementation so that folks can continue to iterate on it (eg. while I'm on vacation until Aug 22nd).

@NavidZ @scottgonzalez @patrickhlauke @teddink WDYT, does this look like a reasonable place to start on the branch?

**Comment by patrickhlauke** [comment]
Looks reasonable to me, yes

**Comment by NavidZ** [comment]
lgtm too.

File: index.html [hunk 1]
```diff
@@ -856,13 +858,16 @@
...
+            <p>Direct manipulation devices should behave exactly as if <a data-lt="Element.setPointerCapture">setPointerCapture</a> was called on the target element just before the invocation of any <code>pointerdown</code> listeners.  The <a data-lt="Element.hasPointerCapture">hasPointerCapture</a> API may be used (eg. within any <code>pointerdown</code> listener) to determine whether this has occurred.  If <a data-lt="Element.releasePointerCapture">releasePointerCapture</a> is not called for the pointer before the next pointer event is fired, then a <a>gotpointercapture event</a> will be dispatched to the target (as normal) indicating that capture is active.
```
Comment (scottgonzalez) [inline index.html]: These two paragraphs are missing closing tags.

**Comment by scottgonzalez** [comment]
One minor nit, but otherwise this looks good.

File: index.html (hunk 1 above)
Comment (RByers) [inline index.html]: Thanks!  Fixed.

**Pull request merged**